app/controllers/substitutions_controller.rb

```diff
@@ -16,13 +16,23 @@ def index
   def new
     @substitution = Substitution.new()
 
-    if params[:mentor_id]
-      @substitution.mentor = Mentor.find(params[:mentor_id])
-      @substitution.kid = @substitution.mentor.kids.first
+    if params[:kid_id]
```

Comment:
I think this can be done easier. Perhaps something like @substitution = Substitution.new(kid_id: params[:kid_id])